### DISCUSSION TITLE
Mouse getting lost in Virtual Display

### DISCUSSION THREAD

**Discussion by willj1220** [VirtualDrivers/Virtual-Display-Driver#186] [Q&A]
Does anyone have any suggestions how to have the Virtual Display enabled, but disallow my mouse to move over to that screen? I keep getting my mouse lost off-screen when I'm not gaming. Thanks in advance!

**Comment by roshkins** [comment]
If you don't need to see the display, you can disable the driver and your
mouse will not be lost.

**Comment by bud3699** [comment]
> Does anyone have any suggestions how to have the Virtual Display enabled, but disallow my mouse to move over to that screen? I keep getting my mouse lost off-screen when I'm not gaming. Thanks in advance!

Like roshkins said, you can just disable the display if you dont need it, or move it to a top corner or something with a tiny slither accesible. I believe with power toys there is also a mouse finder tool, not sure if its something you want to check out

**Comment by ShaunDreclin** [comment]
(Extremely) late response, but an AutoHotKey script to reset your mouse position to your main monitor would be pretty easy to throw together

```ahk
F1::
  MouseMove, 0, 0
return
```
